src/epic/modules.py

```diff
@@ -0,0 +1,17 @@
+# %%
```

Comment:
Remove VS code notebook delimiters once PR is ready

Reply:
Done.